HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -2249,80 +2265,12 @@ def preSolve(self):
         if not self.quiet:
             self.checkConditions(verbose=self.verbose,public_call=False)
 
-    def checkConditions(self,verbose=False,public_call=True):
+    def checkGICInd(self,Thorn,verbose,public_call):
         '''
-        This method checks whether the instance's type satisfies the Absolute Impatience Condition (AIC), Weak Return
-        Impatience Condition (WRIC), Finite Human Wealth Condition (FHWC) and Finite Value of
-        Autarky Condition (FVAC).  When combinations of these conditions are satisfied, the 
-        solution to the problem exhibits different characteristics.  (For an exposition of the
-        conditions, see http://econ.jhu.edu/people/ccarroll/papers/BufferStockTheory/)
-
-        Parameters
-        ----------
-        verbose : boolean
-            Specifies different levels of verbosity of feedback. When False, it only reports whether the
-            instance's type fails to satisfy a particular condition. When True, it reports all results, i.e.
-            the factor values for all conditions.
-
-        Returns
-        -------
-        None
+        Check Individual Growth Impatience Factor.
         '''
-        violated = False # PerfForesightConsumerType.checkConditions(self, verbose=False, verbose_reference=False)
 
-        if self.cycles!=0 or self.T_cycle > 1:
-            return
-
-        # For theory, see hyperlink targets to expressions in 
-        # url=http://econ.jhu.edu/people/ccarroll/papers/BufferStockTheory
-        # For example, the hyperlink to the relevant section of the paper
-        url='http://econ.jhu.edu/people/ccarroll/papers/BufferStockTheory'
```

Comment:
this needs to be there @sbenthall as the variable `url` is referenced multiple times.